openhands/server/routes/public.py

```diff
@@ -104,3 +104,18 @@ async def get_security_analyzers():
         list: A sorted list of security analyzer names.
     """
     return sorted(SecurityAnalyzers.keys())
+
+
+@app.get('/config')
+async def get_config():
+    """
+    Get current config
+    """
+
+    config = {
+        'APP_MODE': openhands_config.APP_MODE,
+        'GITHUB_CLIENT_ID': openhands_config.GITHUB_CLIENT_ID,
+        'POSTHOG_CLIENT_KEY': openhands_config.POSTHOG_CLIENT_KEY,
+    }
```

Comment:
An alternative could be a method on the class that returns this dict, instead of us writing them manually here